config/secrets.json.example

```diff
@@ -1,91 +0,0 @@
-{
```

Comment:
Why was this file deleted? Based on the current README, developers use this file as the baseline for their `secrets.json`